tests/console/system_prepare.pm

```diff
@@ -44,12 +44,18 @@ sub run {
     }
 
     # Register the modules after media migration, so it can do regession
-    if (get_var('SCC_ADDONS') && get_var('MEDIA_UPGRADE') && get_var('KEEP_REGISTERED')) {
+    if (get_var('SCC_ADDONS') && get_var('MEDIA_UPGRADE') && get_var('REGISTER_AFMIG')) {
```

Comment:
Maybe we can set the variable like 'RE_REGISTER_AFTER_UPGRADE', WDYT?

Reply:
A little long. Change to REREG_AF_UPG OK?   Don't use too long name